services/api/src/resources/user/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import { Pool } from 'mariadb';
+import { query } from '../../util/db';
+import { Sql } from './sql';
+// import { logger } from '../../loggers/logger';
+
+export const Helpers = (sqlClientPool: Pool) => {
```

Comment:
The models were created to supersede the helper files pattern. All this should be moved into to the User model (part of, or called by, `UserModel.deleteUser`) which should be the single place for business logic.

Reply:
rgr